andela/ah-shakas-frontend
# #160577437 implement signup

## What does this PR do?
- It implements signup

## Description of Task to be completed?
- To be able to use the application and access authenticated routes, users should be able to Signup and or Register using Sign up forms on the signup route.

## How should this be manually tested?
- Clone this branch ft-signup-160577437
- Run the following commands on your terminal:
  `yarn install` - to install required dependencies
  `yarn start` - to start server
- When the server starts, you will be redirected to the browser
- Navigate to this url: [http://localhost:3000/signup](http://localhost:3000/signup)
- The sign up page should appear as below:
![image](https://user-images.githubusercontent.com/34505753/48153700-f02b0f80-e2d7-11e8-85a2-910568030446.png)
- The following screenshot is an illustration of how error messages will be displayed:
![image](https://user-images.githubusercontent.com/34505753/48154020-abec3f00-e2d8-11e8-8939-fb030e16fbd0.png)

**Tests**
- Run `yarn coverage` or `yarn test` on your terminal

## Any background context you want to provide?
- After successful signup, you should see a message prompting you to check your email for verification

## What are the relevant pivotal tracker stories?
[160577437](https://www.pivotaltracker.com/n/projects/2198533/stories/160577437)

## Deployment note
N/A

## Related PRs branch | PR (branch_name) | (pr_link)
[ft-signup-160577437](https://github.com/andela/ah-shakas-frontend/tree/ft-signup-160577437)

## Todos
